tests/countries/test_syrian_arab_republic.py

```
@@ -0,0 +1,145 @@
+            "2022-05-04",
+            "2023-04-21",
+            "2023-04-22",
+            "2023-04-23",
+            "2024-04-10",
+            "2024-04-11",
+            "2024-04-12",
+        )
+         self.assertHolidayName(name, self.no_estimated_holidays, dts)
+         self.assertHolidayName(name, self.no_estimated_holidays, range(1940, 2050))
+
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)